### ISSUE TITLE
Skeleton should have accessible text

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#4997]
**Describe the issue. What is the expected and unexpected behavior?**
When items are still loading, we provide the visual indicator to sighted users with the skeleton component. We should also provide some kind of text for screen reader users to let them know that a section is still loading or will ideally soon have content. The approach I'd suggest is the one that [OpenShift](https://github.com/openshift/console/pull/6743/files) has taken with their status section skeletons with screen reader only text. 

I imagine we could create a prop that would determine the text inside the span (that wouldn't be required). We could create a sensible default, but I am curious what others think about the scenario of using multiple skeleton components on a page. Perhaps you would only use the prop/screen reader text on the first one? I wouldn't want to overload screen reader users with redundant information like seeing "Loading, loading, loading" a bunch of times if there's multiple skeletons.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

Open VO or another screen reader with the skeleton component. It will skip right over it.

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug - it's currently not perceivable by screen reader
